src/Form/selectors.js

```diff
@@ -0,0 +1,24 @@
+const EMAIL_SELECTOR = `
```

Comment:
I've moved all selectors in here, so they're easy to re-use and make less noise in the other files.